code/modules/mob/living/carbon/carbon.dm

```diff
@@ -89,6 +89,8 @@
 
 
 /mob/living/carbon/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, override = 0, tesla_shock = 0)
+	if(dna && dna.check_mutation(SMES_HUMAN))

```

Comment:
Yes, but then species change would remove any shock protection from the mutation. A good wy to get +1 issue nobody would ever fix.